VikingQuiz.UI/src/Components/MyQuizesPage/ManageQuizComponent/ManageQuizComponent.css

```diff
@@ -0,0 +1,46 @@
+.manage_Quiz{
+    background-color: #91becf;
+    border-radius: 50%;
+    width: 110px;
+    height: 110px;
+    margin-left: 30%;
+    margin-top: 20px;
+    position: relative; 
+    border:none;
```

Comment:
Please format the code. There should be a space between the column ( ':' ) and the value for the property.